apps/condo/pages/index.tsx

```diff
@@ -1,33 +1,45 @@
-import Head from 'next/head'
-import { Typography } from 'antd'
 import React from 'react'
-import ReactMarkdown from 'react-markdown'
-
+import Head from 'next/head'
+import { Typography, Space } from 'antd'
 import { useIntl } from '@core/next/intl'
 
 import { PageContent, PageHeader, PageWrapper } from '@condo/domains/common/components/containers/BaseLayout'
 import { OrganizationRequired } from '@condo/domains/organization/components/OrganizationRequired'
+import { TicketsWidget } from '@condo/domains/ticket/components/TicketsWidget'
+
 
-const IndexPage: React.FC = () => {
+const IndexPage = () => {
     const intl = useIntl()
     const PageTitleMsg = intl.formatMessage({ id: 'pages.index.PageTitle' })
-    const WeAreStillDevelopingThisPageMsg = intl.formatMessage({ id: 'WeAreStillDevelopingThisPage' })
 
-    return <>
-        <Head>
-            <title>{PageTitleMsg}</title>
-        </Head>
-        <PageWrapper>
-            <PageHeader title={PageTitleMsg}/>
-            <OrganizationRequired>
-                <PageContent>
-                    <Typography.Paragraph>
-                        <ReactMarkdown source={WeAreStillDevelopingThisPageMsg}/>
-                    </Typography.Paragraph>
-                </PageContent>
-            </OrganizationRequired>
-        </PageWrapper>
-    </>
+    return (
+        <>
+            <Head>
+                <title>{PageTitleMsg}</title>
+            </Head>
+            <PageWrapper>
+                <PageHeader style={{ background: 'transparent' }} title={<Typography.Title>{PageTitleMsg}</Typography.Title>}/>
+                <OrganizationRequired>
+                    <PageContent>
+                        <TicketsWidget />
+                    </PageContent>
+                </OrganizationRequired>
+            </PageWrapper>
+        </>
+    )
 }
 
+const HeaderAction = () => {
+    const intl = useIntl()
+    return (
+        <Space>
+            <Typography.Text style={{ fontSize: '12px' }}>
+                {intl.formatMessage({ id: 'menu.Analytics' })}
```

Comment:
Translations should be at the top of the component right after useIntl, so extract const please...

Reply:
Fixed